tests/test_search_runs.py

```diff
@@ -0,0 +1,43 @@
+from click.testing import CliRunner
+from unittest import mock
+
+import os
+import numpy as np
+import pandas as pd
+import shutil
+import tempfile
+import textwrap
+
+from mlflow import experiments
+
+
+def test_csv_generation():
```

Comment:
This test case description doesn't really match the suite / module name (`test_search_runs`), since it's not really testing `search_runs` behavior. Can we move this back to `test_runs` and do a `skipif skinny` ?